protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectdata/ArrayDataConverter.java

```diff
@@ -0,0 +1,30 @@
+package com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectdata;
+
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.Message;
+import org.apache.kafka.connect.data.Schema;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class ArrayDataConverter implements DataConverter {
+
+    @Override
+    public void toProtobufData(
+            final Schema schema,
+            final Object value,
+            final Descriptors.FieldDescriptor fieldDescriptor,
+            final Message.Builder messageBuilder) {
+
+        PrimitiveDataConverter primitiveDataConverter = new PrimitiveDataConverter();
```

Comment:
Can this be created once per class? Creating once per call is not idea.

Reply:
Is the data in the array conversion case always going to be primitive?

Reply:
@allkliu Good catch. It can be a complex type like message. But since we haven't implemented the message conversion, we will need to refactor and test it in the future.

Reply:
@blacktooth I will refactor it to be created per class.